node/coinstacks/gnosis/daemon/init.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+set -e
+
+start_coin_bg() {
+  ./Nethermind.Runner --config gnosis --datadir /data/gnosis --JsonRpc.Host=0.0.0.0 --JsonRpc.Port=8545 --JsonRpc.WebSocketsPort=8546 --JsonRpc.JwtSecretFile=/jwt.hex --Network.ActivePeersMaxCount=100 --HealthChecks.Enabled=true --Metrics.CountersEnabled=true 
```

Comment:
we need the debug module for tracing with current logic: 
`--JsonRpc.EnabledModules=eth,net,web3,debug,subscribe,trace,txpool,health,rpc`

also let's remove the `ActivePeersMaxCount` it is listed as deprecated and no need to mess with peer count unless we are experienceing sync issues

Reply:
Ad 2. Used the wrong prop but here's the reasoning for bumping it up:

```
if you have a good connection and a reasonable machine then setting --Network.MaxActivePeers to 256 (see NetworkConfig secion in configuration) should give much better fast sync times (we use 256 peers to get 5 hours syncs)
```
https://docs.nethermind.io/nethermind/ethereum-client/sync-modes